### ISSUE TITLE
Crash When Creating world / Opening World With Mod

### ISSUE THREAD

**Issue by DrakeOwenIO** [rwtema/extrautilities_old_issues#510]
I am having an issue when creating or opening a world with extrautilities. I crash every time I have ran the mod by itself and with my other mods. I have ran it with and without optifine. I am very confused about why it will not work. Please help because I really want to play Minecraft using your mod. The crash report is below.

http://pastebin.com/aXbGJ6YG

**Comment by TherminatorX** [comment]
It's an known interaction bug with Rei's Minimap. Tema doesn't seem interested in fixing it, but it's likely on his end.

Umpteenth dupe of #406 
Just switch minimaps for now, and close the issue.

**Comment by Toerr** [comment]
I got a similar error with Rei's minimap as the OP. Then I switched to Journey Map and got a new error:

http://pastebin.com/UVGSANTu

Couldn't really make heads or tails of it, but it basically does the same thing as REI's (Terrain Builds but internal server shuts down before the world is loaded) Anyway, would very much like a minimap so any help would be appreciated. 
Thanks :]

**Comment by TherminatorX** [comment]
@Toerr The REI problem is a stack overflow error that actually mentions a mod. This one only references vanilla problems.

**Comment by Toerr** [comment]
Yeah, got help on MCF.net, just had to downgrade JM to 4.0.4. Thanks for the reply though.
